src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -19,13 +19,17 @@ class ElectronicDeliveryForm extends React.Component {
     this.state = {
       form: !_isEmpty(this.props.form) ? this.props.form :
         {
-          name: '',
-          email: '',
+          emailAddress: '',
```

Comment:
That's fine. It was more of having those empty initial settings so that the form could be filled out by React.